contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -374,9 +394,9 @@ and should assume an Open World.
 An example of an oscillating condition type is `Ready` (despite it running
 afoul of current guidance), which indicates the object was believed to be fully
```

Comment:
if this changes the guidance around positive/negative conditions, we can drop the `(despite it running afoul of current guidance)` parenthetical

Reply:
Thanks for catching this, updated.